src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react'
+import { Container, Title, Logout, Subtitle } from './styles'
+import { signOut } from '../../service/authentication'
+import PlaylistItem from './PlaylistItem'
+import { playlistsAsyncActions } from '../../store/slices/playlists/slice'
+import { useDispatch } from 'react-redux'
+import { selectCurrentUser } from '../../store/slices/authentication/selectors'
+import { selectOwnPlaylists } from '../../store/slices/playlists/selectors'
+import { useSelector } from 'react-redux'
+import PlaylistData from '../../types/PlaylistData'
+
+const Sidebar = () => {
+
+    const dispatch = useDispatch()
+    const currentUser = useSelector(selectCurrentUser)
+    const ownPlaylists = useSelector(selectOwnPlaylists)
+
+    useEffect(() => {
+        dispatch(playlistsAsyncActions.getCurrentUserPlaylists(currentUser.id))
+    }, [dispatch, currentUser])
+
+    const handleLogout = () => {
+        signOut()
+    }
+    return (
+        <Container>
+            <Title>Collabox</Title>
+            <Subtitle>My playlists</Subtitle>
+            {ownPlaylists?.map((playlist: PlaylistData) => {
+                return <PlaylistItem key={playlist.id} playlist={playlist} />
+            })}
+            <Logout onClick={handleLogout}>logout</Logout>
+        </Container>
+    )
```

Comment:
I would add the Signput button to the top, because this list can get really long. Also, users usually look for signout on the top.

Reply:
i wasnt sure, where to put it, i dont think it belongs to the sidebar, so i think i wait untill the app develops further and then decide where to put it